src/services/userService.js

```diff
@@ -215,24 +215,34 @@ async function returnAllFavouriteEvents(userId) {
     }
 }
 
-async function approvedUser(userId, requestBody) {
+async function approveOrganizer(userId, requestBody) {
     const existingUser = await User.findById(userId);
+    if (existingUser.isDeleted == true) {
```

Comment:
As discussed, added it directly in the find function.

Reply:
From an error handling perspective, I've written it this way.